src/main/java/mServer/crawler/sender/phoenix/tasks/PhoenixFilmDetailTask.java

```diff
@@ -0,0 +1,147 @@
+package mServer.crawler.sender.phoenix.tasks;
+
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mlib.daten.DatenFilm;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.net.MalformedURLException;
+import java.time.format.DateTimeFormatter;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import javax.ws.rs.client.WebTarget;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import mServer.crawler.CrawlerTool;
+import mServer.crawler.sender.MediathekReader;
+import mServer.crawler.sender.newsearch.Qualities;
+import mServer.crawler.sender.orf.CrawlerUrlDTO;
+import mServer.crawler.sender.orf.tasks.AbstractRecursivConverterTask;
+import mServer.crawler.sender.phoenix.DownloadDto;
+import mServer.crawler.sender.phoenix.PhoenixConstants;
+import mServer.crawler.sender.phoenix.parser.PhoenixFilmDetailDeserializer;
+import mServer.crawler.sender.phoenix.parser.PhoenixFilmDetailDto;
+import mServer.crawler.sender.phoenix.parser.PhoenixFilmXmlHandler;
+import mServer.crawler.sender.phoenix.parser.ZdfDownloadDtoDeserializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.xml.sax.SAXException;
+
+public class PhoenixFilmDetailTask extends ZdfTaskBase<DatenFilm, CrawlerUrlDTO> {
+
+  private static final Logger LOG = LogManager.getLogger(PhoenixFilmDetailTask.class);
+
+  private static final Type OPTIONAL_FILM_DETAIL_DTO_TYPE_TOKEN = new TypeToken<Optional<PhoenixFilmDetailDto>>() {
+  }.getType();
+  private static final Type OPTIONAL_DOWNLOAD_DTO_TYPE_TOKEN = new TypeToken<Optional<DownloadDto>>() {
+  }.getType();
+
+  private static final DateTimeFormatter DATE_FORMAT
+          = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+  private static final DateTimeFormatter TIME_FORMAT
+          = DateTimeFormatter.ofPattern("HH:mm:ss");
+
+  private final String filmDetailHost;
+  private final String videoDetailHost;
+
+  public PhoenixFilmDetailTask(MediathekReader aCrawler,
+          ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDTOs, Optional<String> aAuthKey,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDTOs" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)